libs/domain/search/facet/facet.component.ts

```diff
@@ -58,6 +57,7 @@ export class SearchFacetComponent extends I18nMixin(LitElement) {
       .heading=${this.name}
       .selectedLength=${selectedLength}
       .valuesLength=${valuesLength}
+      .key=${featureVersion >= '1.4' ? facet.parameter : undefined}
```

Comment:
ifDefined?

Reply:
not needed. This is part of property binding anyways, and the attribute is not reflected.